### PR TITLE
Reworking the document to focus on what needs to be done

### PR DESCRIPTION
**Pull request by Leena**
Could you review these edits to the README please? 
I was hoping it could better show the intentions of the group.

### CODE REVIEW

**Comment by Leena** [comment]
You can see the edits in markdown [here](https://github.com/DanielCosme/Launch-School-Advanced-JavaScript-Discussion-Group/pull/1/commits/6732433d504ca3ee5db99db1e5834da3ee5998e1#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5).

File: README.md [hunk 1]
```diff
@@ -1,31 +1,31 @@
...
-Join us for a round table of analyzing, critical thinking and creating with JavaScript! Think of this as a way to test everything you've learned so far and use creativity as a tool to deepen your fundamentals.
+Join us for an hour of of analyzing, critical thinking and creating with JavaScript! Think of this as a way to test everything you've learned so far and use creativity as a tool to deepen your fundamentals.
 ## Pre-requisites:
-- JavaScript Track: JS130, Up to Testing
+- JavaScript Track: JS130
 - Ruby Track: JS225
-- Understand Bloom's [Taxonomy Launch School Presentation](https://www.reddit.com/r/launchschool/comments/kvf6pv/blooms_taxonomy_x_launch_school/), [General introduction](https://tophat.com/blog/blooms-taxonomy/) and [List of Questions](https://www.eoas.ubc.ca/research/cwsei/resources/Blooms-verbs-stems.htm).
+- Check out Bloom's Taxonomy with one of these: [Launch School Presentation](https://www.reddit.com/r/launchschool/comments/kvf6pv/blooms_taxonomy_x_launch_school/), [General introduction](https://tophat.com/blog/blooms-taxonomy/) and [List of Questions](https://www.eoas.ubc.ca/research/cwsei/resources/Blooms-verbs-stems.htm).
```
Comment (katarinarosiak) [inline README.md]: How about: 'Check out Bloom's Taxonomy for inspiration with one of these:' or 'Check out Bloom's Taxonomy for learning more about how to deepen your knowledge of a material'

Comment (Leena) [reply README.md]: Love it! Great idea @katarinarosiak !

File: README.md [hunk 2]
```diff
@@ -1,31 +1,31 @@
...
+- Practice intentional problem solving collaboratively 
+- Practice technical communication and precision
+- Practice levels 4,5 and 6 of the Taxonomy
 ## Guidelines for Participants
-If you feel comfortable with a question, make space for others first, then help them fill their gaps
-This is a space for everyone to contribute, participate, take risks, experiment,  to be open-minded to learning and LS Code of Conduct and Community Guidelines. 
+- If you feel comfortable with a question, make space for others first, then help them fill their gaps. 
+- This is a space for everyone to contribute, participate, take risks and experiment. Be open-minded, willing to learn and respect the LS Code of Conduct and Community Guidelines. 
+- Everyone should lead the session at least once.
+- Weekly rotation of leader.
```
Comment (katarinarosiak) [inline README.md]: Maybe better: 'We encourage everyone to lead a session at least once'  or 'Everyone can be a leader for this group, as long as you have an interesting problem to solve or discussion topic to share'. 'should' may sound a bit harsh. What do you think?

Comment (Leena) [reply README.md]: Yes, I think the tone is better the way you write it.

**Review by katarinarosiak — APPROVED** [review]

**Pull request merged**